### PR TITLE
Created basic ranged enemy

### PR DESCRIPTION
**Pull request by ethanakester**

### CODE REVIEW

File: Assets/Scripts/Enemy/EnemyFactory.cs [hunk 1]
```diff
@@ -51,6 +51,7 @@ public EnemyFactory()
         // Define enemyInfo for each type.
         AddTurretToRoster();
         _enemyInfo.Add(EnemyType.GRENADIER, new EnemyInfo(EnemyMovements.GrenadierMovement, Globals.DirectTargeting, EnemyMovements.GrenadierSetup, new ArtilleryAttackBehaviour(EntityType.ENEMY, 20f), 10f));
+        _enemyInfo.Add(EnemyType.RANGED, new EnemyInfo(EnemyMovements.RangedMovement, Globals.DirectTargeting, EnemyMovements.RangedSetup, new BulletAttackBehaviour(EntityType.ENEMY, 5f, 20f), 1f));
```
Comment (MaXeraph) [inline Assets/Scripts/Enemy/EnemyFactory.cs:54]: Are we gonna be implementing enemies like this or do we want to put it in a method so its easier to read ?

File: Assets/Scripts/Globals.cs [hunk 2]
```diff
@@ -41,15 +41,17 @@ public static Vector3 DirectTargeting(Transform from, Transform to)
     {
         { EnemyType.TURRET, "Turret" },
         { EnemyType.HEALER, "Healer" },
-        { EnemyType.GRENADIER, "Grenadier"}
+        { EnemyType.GRENADIER, "Grenadier"},
+		{ EnemyType.RANGED, "Ranged"}
```
Comment (MaXeraph) [inline Assets/Scripts/Globals.cs:45]: extra space before the {RANGED, "Ranged"}

File: Assets/Scripts/Globals.cs [hunk 3]
```diff
@@ -41,15 +41,17 @@ public static Vector3 DirectTargeting(Transform from, Transform to)
...
     };
 }
 
 public enum EnemyType
 {
 	TURRET,
 	GRENADIER,
-  HEALER
+	RANGED,
+	HEALER
```
Comment (MaXeraph) [inline Assets/Scripts/Globals.cs:54]: We dont have Healer type

File: Assets/Scripts/Globals.cs (hunk 2 above)
Comment (zhuandy42) [reply Assets/Scripts/Globals.cs]: spacing!!!

File: Assets/Scripts/Globals.cs [hunk 4]
```diff
@@ -41,15 +41,16 @@ public static Vector3 DirectTargeting(Transform from, Transform to)
     {
         { EnemyType.TURRET, "Turret" },
         { EnemyType.HEALER, "Healer" },
```
Comment (MaXeraph) [inline Assets/Scripts/Globals.cs:43]: This too

File: Assets/Scripts/Enemy/EnemyFactory.cs (hunk 1 above)
Comment (ethanakester) [reply Assets/Scripts/Enemy/EnemyFactory.cs:54]: yes

File: Assets/Scripts/Globals.cs (hunk 4 above)
Comment (ethanakester) [reply Assets/Scripts/Globals.cs:43]: we will need something like this for variants and their prefabs

**Review by zhuandy42 — APPROVED** [review]

**Review by MaXeraph — APPROVED** [review]

**Pull request merged**